packages/marko-web-native-x/utils/create-client.js

```diff
@@ -0,0 +1,14 @@
+const fetch = require('node-fetch');
```

Comment:
Move this to an `apollo` folder so it's more clear what kind of client is being created